Cargo.toml

```diff
@@ -28,31 +28,29 @@ maintenance = { status = "actively-developed" }
 # Disable CORS Check
 unsafe-disable-cors = []
 
+# Enable Logging
+log = []
+
 [dependencies]
-ark-serialize = { version = "0.3.0", default-features = false }
-async-std = { version = "1.10.0" }
-bip0039 = { version = "0.10.0" }
-bip32 = { version = "0.2.2" }
-bs58 = { version = "0.4.0" }
-cocoon = { version = "0.3.0" }
-codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = [ "derive", "full" ] }
+argon2 = { version = "0.3.4", default-features = false, features = ["alloc", "password-hash"] }
+chrono = { version = "0.4.19", default-features = false, features = ["clock"] }
 derivative = { version = "2.2.0", default-features = false, features = ["use_core"] }
 dirs-next = { version = "2.0.0" }
-futures = { version = "0.3.17", default-features = false }
+futures = { version = "0.3.17", default-features = false, features = ["alloc"] }
 http-types = { version = "2.12.0", default-features = false }
-manta-api = { git = "https://github.com/Manta-Network/manta-api", branch = "manta", default-features = false, features = ["signer", "testnet"] }
-manta-asset = { git = "https://github.com/Manta-Network/manta-types", branch = "manta" }
-manta-crypto = { git = "https://github.com/Manta-Network/manta-crypto", branch = "manta" }
-manta-data = { git = "https://github.com/Manta-Network/manta-types", branch = "manta" }
-rand = { version = "0.8.4", default-features = false }
-rand_chacha = { version = "0.3.1", default-features = false }
+manta-accounting = { path = "../manta-rs/manta-accounting", features = ["cocoon-fs"] }
+manta-crypto = { path = "../manta-rs/manta-crypto", features = ["getrandom"] }
+manta-pay = { path = "../manta-rs/manta-pay", features = ["groth16", "serde", "wallet"] }
+manta-sdk = { path = "../sdk", features = ["download"] }
+manta-util = { path = "../manta-rs/manta-util" }
```

Comment:
Looks like the e2e is failing because of relative paths.

Reply:
Also, the node and e2e test haven't been updated, so it will fail anyway.